src/main/java/org/jboss/snowdrop/licenses/xml/DependencyElement.java

```diff
@@ -161,4 +166,19 @@ public int hashCode() {
         result = 31 * result + version.hashCode();
         return result;
     }
+
+    public String toGav() {
```

Comment:
Maybe rename to `toGavString` as there is a class called `Gav` this might be confusing.

Reply:
Done